src/System.Private.CoreLib/shared/System/Diagnostics/Debug.Unix.cs

```diff
@@ -21,6 +21,8 @@ private static void ShowAssertDialog(string stackTrace, string message, string d
                 // In Core, we do not show a dialog.
                 // Fail in order to avoid anyone catching an exception and masking
                 // an assert failure.
+
+                // We can safely ignore errorSource since it's a CoreCLR specific argument for distinguishing calls from Debug.Assert and Environment.FailFast
```

Comment:
This file is used on CoreCLR on Unix today. I do not think we should be dropping `errorSource` here - we want a good failfast message in CoreCLR on Unix too.